app-steady/app-steady/SecondViewController.swift

```diff
@@ -7,12 +7,40 @@
 //
 
 import UIKit
+import Charts
 
 class SecondViewController: UIViewController {
+    
+    var lineChart = LineChartView(frame: CGRectMake(100, 100, 400, 400))
+    
+    var days: [String]!
 
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
+        
+        self.initializeLineChart()
+    }
+    
+    func initializeLineChart(){
+        days = ["Mon", "Tues", "Wed", "Thurs", "Fri", "Sat", "Sun"]
+        let rank = [4.0, 6.0, 8.0, 7.0, 10.0, 10.0, 8.0]
+        setChart(days, values: rank)
```

Comment:
If I write the line as `setChart(dataPoints: days, values: rank)` I get an error just the same as if i attempt to write it as `setChart(days, rank)` it may just be convention but Xcode will take out `dataPoints` as the fix